jira/client.py

```diff
@@ -92,7 +92,8 @@ def emit(self, record):
 #    "which means that there is a big change of having problems. " \
 #    "Possible workaround http://stackoverflow.com/a/17628350/99834" % encoding)
 
-logging.getLogger('jira').addHandler(NullHandler())
+logger = logging.getLogger('jira')
```

Comment:
In this case I vote for using `__name__`, because
* most popular option among serious packages
* backwards compatible with "jira" (if someone it change jira logger, the change would also apply to jira.client because is a child one).
* allows separation of logging if someone is renaming the main package or if it embeds jira package in their own module.

The logging cookbook could use a pull request, just to clarify the recommended way of using logging in libraries. I bet we are not the only ones confused by it.